storybookjs/storybook
# [WIP] Add Polymer 2 support

## What I did

We added initial support for Polymer 2 and set up an example application. We are in the process of improving the compatibility in all browsers and adding support for all Storybook addons. Tests have to be added.

## How to test
* Run `yarn` in the repo root
* Run `yarn bootstrap` in the repo root
* Change to `examples/polymer-cli` directory (`cd examples/polymer-cli`)
* Start Storybook (`yarn storybook`)

The Welcome page and example stories should be shown at `http://localhost:9001/`

## TODO
- [x] Firefox support
- [x] Fix `getstorybook` CLI command for polymer 
- [x] Support for other addons, especially `knobs`
- [x] Improve examples stories
- [x] Allow string templates
- [x] Add example stories for Polymer 
- [x] HMR
- [x] Try to generalise for all web component frameworks (won't fix in this PR)
- [ ] Cleanup code (lot of React stuff in there from copying)
- [x] Bring PR up to date with latest changes
